packages/@aws-cdk/aws-eks/README.md

```diff
@@ -50,7 +50,8 @@ This example defines an Amazon EKS cluster with the following configuration:
 ```ts
 // provisiong a cluster
 const cluster = new eks.Cluster(this, 'hello-eks', {
-  version: eks.KubernetesVersion.V1_21,
+  version: eks.KubernetesVersion.V1_23,
+  kubectlLayer: new KubectlV23Layer(this, 'kubectl'),
```

Comment:
Very soon the latest will be 1_24 :). Thats already in the main branch and waiting to be released next week. Lets update to that